xml/System/DateTime.xml

```diff
@@ -79,54 +89,80 @@
   
 -   By calling any of the overloads of the <xref:System.DateTime> constructor that allow you to specify specific elements of the date and time value (such as the year, month, and day, or the number of ticks). The following statement illustrates a call to one of the <xref:System.DateTime> constructors to create a date with a specific year, month, day, hour, minute, and second.  
   
-     [!code-csharp[System.DateTime.Instantiation#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.DateTime.Instantiation/cs/Instantiation1.cs#1)]
-     [!code-vb[System.DateTime.Instantiation#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Instantiation/vb/Instantiation1.vb#1)]  
-  
+    # [Visual Basic](#tab/visual-basic)
+    [!code-vb[System.DateTime.Instantiation#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Instantiation/vb/Instantiation1.vb#1)]  
+    # [C#](#tab/csharp)
+    [!code-csharp[System.DateTime.Instantiation#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.DateTime.Instantiation/cs/Instantiation1.cs#1)]
+    ---
+
 -   By using any compiler-specific syntax for declaring date and time values. For example, the following Visual Basic statement initializes a new <xref:System.DateTime> value.  
   
-     [!code-vb[System.DateTime.Instantiation#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Instantiation/vb/Instantiation1.vb#2)]  
-  
+    # [Visual Basic](#tab/visual-basic)
+    [!code-vb[System.DateTime.Instantiation#2](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.Instantiation/vb/Instantiation1.vb#2)]  
+    ---
```

Comment:
Should there be a C# equivalent for this?

Reply:
Nope, this is only a VB thing.